src/sentry/static/sentry/app/views/organizationContext.jsx

```diff
@@ -155,14 +158,26 @@ const OrganizationContext = createReactClass({
           GlobalSelectionStore.loadInitialData(data, this.props.location.query);
         }
         OrganizationEnvironmentsStore.loadInitialData(environments);
-
-        this.setState({
-          organization: data,
-          loading: false,
-          error: false,
-          errorType: null,
-          hooks,
-        });
+        this.setState(
+          {
+            organization: data,
+            loading: false,
+            error: false,
+            errorType: null,
+            hooks,
+          },
+          () => {
+            // Take a measurement for when organization details are done loading and the new state is applied
+            metric.measure({
+              name: 'app.component.organization-details-fetch',
```

Comment:
1) we should name this something a bit more generic because...
2) we have to add this name to an allow-list in [reload](https://github.com/getsentry/reload/blob/master/reload_app/metrics/__init__.py#L4) for this to work

maybe something like `app.component.perf`?